promql/promqltest/testdata/functions.test

```diff
@@ -1239,11 +1240,16 @@ clear
 load 5m
 	exp_root_log{l="x"} 10
 	exp_root_log{l="y"} 20
+    exp_root_log_h{l="z"} {{schema:1 sum:1 count:1}}
```

Comment:
This line uses spaces, but the previous lines use tabs. Depending on the tab width, that might lead to weird alignment. (E.g. GH displays this file with 8-space tabs, so the lines above are too far indented.)

I recommend to use a tab in this line too. (Or change all lines to use spaces.)